### ISSUE TITLE
check if one of the aliases is banned

### ISSUE THREAD

**Issue by Smorted** [JavierLeon9966/Alias#10]
can you make it check if one of the aliasses is banned it kicks the player with the same "you are banned" message? 

so instead of only letting us know that this player has alternative accounts give us also the option to automatically re ban him again.

**Comment by JavierLeon9966** [comment]
That option already exists:
https://github.com/JavierLeon9966/Alias/blob/9aa46fcf74064ff6e26c38fb8a2adf06d9a27562/resources/config.yml#L19-L20

**Comment by Smorted** [comment]
no this option doesn't check if the player alias is inside the banned-player.txt, it just detects if the player has an alternative account and if so it directly kicks him

what if i have two accounts? if i join using the second one i will be directly kicked

**Comment by JavierLeon9966** [comment]
> no this option doesn't check if the player alias is inside the banned-player.txt, it just detects if the player has an alternative account and if so it directly kicks him
> 
> what if i have two accounts? if i join using the second one i will be directly kicked

So you want an option to kick players whose alternative account was banned by Pocketmine instead of banning players that have alternative accounts?

**Comment by Smorted** [comment]
yes exactly, just add it as an optional feature because this way we can identify banned players and actually automatically reban them, perfect ban system!
